src/tike/operators/__init__.py

```diff
@@ -9,9 +9,10 @@
 better operators in the future.
 
 All operator methods should take NumPy arrays as inputs. This is a design
-decision which was made because clients of the operators library should not need
-to be concerned about memory locality which is necessary complexity when
-implementating operators for specialized hardware (such as GPUs).
+decision which was made because clients of the operators library should not
+need to be concerned about memory locality which is necessary complexity when
+implementating operators for specialized hardware (such as GPUs). Additionally,
+we assume that GPU memory is too small to hold an entire workload at once.
```

Comment:
...memory is often too small...

Reply:
This docstring is actually obsolete with #46, because Viktor convinced me that it's faster to do extra computation on the GPU and minimize the memory footprint than it is to cache intermediate results in RAM and then need to transfer data between host and GPU.